ds004129-pheno_segregated/sub-ON66199/phenotype/bai.tsv

```diff
@@ -0,0 +1,2 @@
+participant_id	bai01_r03, bai1_1	bai02_r03, bai1_2	bai03_r03, bai1_3	bai04_r03, bai1_4	bai05_r03, bai1_5	bai06_r03, bai1_6	bai07_r03, bai2_7	bai08_r03, bai2_8	bai09_r03, bai2_9	bai10_r03, bai2_10	bai11_r03, bai2_11	bai12_r03, bai3_12	bai13_r03, bai3_13	bai14_r03, bai3_14	bai15_r03, bai3_15	bai16_r03, bai3_16	bai17_r03, bai4_1	bai18_r03, bai4_2	bai19_r03, bai4_3	bai20_r03, bai4_4	bai21_r03, bai4_5
+sub-ON66199	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
```

Comment:
I can't remember if we decided since this file is nested under a participant_id folder that we don't need the participant_id column, or if maybe it just always needs to be there. Open to commentary...